### ISSUE TITLE
`Constants` import failure on build runner generation

### ISSUE THREAD

**Issue by bZichett** [simolus3/moor#2423]
I am pretty sure a minor but breaking drift_dev error started occurring after upgrading new `build_runner` or `source_gen` after flutter 3.10 / dart 3 support was rolled out a few days back.  After building, the generated files have altered imports (`Constants` top level instead of ex: `i3.Constants`) and the (edit: run time) compilation fails. I can try to give more details but curious if anyone else has run into this yet.

**Comment by simolus3** [comment]
I can reproduce the issue and I know how to fix it, but I need to check which AST nodes could be affected by this. I hope I can get a fix out tomorrow!

As a temporary workaround, it might work a bit better if you temporarily downgrade to `analyzer: '>=5.11.0 <5.12.0'` by adding it to your `dependency_overrides` if that doesn't cause another dependency conflict in your app.

**Comment by bZichett** [comment]
Thanks for the speedy answer and suggestion; just wanted to make sure it wasn't something specific to what I did since other possibilities existed as I upgraded to 3.10.

> I hope I can get a fix out tomorrow!

Not urgent at all, don't rush.  Thanks again for your supreme work on this lib, Simon 🙏

**Comment by simolus3** [comment]
I think I've fixed the problematic cases in the linked commit, you can try it out by adding this to your pubspec (and removing the analyzer override):

```yaml
dependency_overrides:
  drift_dev:
    git:
      url: https://github.com/simolus3/drift.git
      ref: 1d63ccb3f145b5029b6356cf9d8fe03f0aefe4ec
      path: drift_dev
```

**Comment by simolus3** [comment]
Should be fixed in `drift_dev: ^2.8.1`.

**Comment by bZichett** [comment]
Works perfectly here :+1:
